sktime/registry/_tags.py

```diff
@@ -237,6 +237,7 @@
         "bool",
         "contract time setting, does the estimator support limiting max fit time?",
     ),
+    # todo, deprecate. We have base class for this now.
```

Comment:
add version please

Reply:
Do we have the capability to deprecate tags now? I was waiting for that, and just left a note so I didn't forget.

Reply:
yes! See the developer guide, deprecation

Reply:
https://www.sktime.org/en/stable/developer_guide/deprecation.html#deprecating-tags

Reply:
Added the tag to the deprecation dictionaries, I assume that's all that is required for now?